web/source/includes/blank-module.ts

```diff
@@ -0,0 +1,2 @@
+//@ts-ignore
+window['module'] = {}; // Needed for some node-oriented modules to load properly.
```

Comment:
I am very uncomfortable with this. What about websites that have defined their own `module` on `window`? We should not be polluting the global namespace.

Reply:
That'd be... a bit ill-advised, seeing how node is all about using `module` for `module.exports` and such.  (This was necessary to use the `events` module.)

I probably should have at least done `= window['module'] || {};`, though.

Reply:
> That'd be... a bit ill-advised, seeing how node is all about using `module` for `module.exports` and such. (This was necessary to use the `events` module.)

We don't control what other websites do. A web dev is well within their rights to define `window.module='foo';` Node is kinda irrelevant here because this is loaded in the browser.

I remain uncomfortable with this.

Reply:
... and found a smaller version that doesn't suffer from this issue.  Found it next to `events` in the `node_modules` folder, actually - guess it's used by one of our dev-dependencies.

Full resolution of this detail incoming soon.